.gitattributes

```diff
@@ -1 +1,4 @@
-*.js linguist-language=TypeScript
\ No newline at end of file
+*.js linguist-language=TypeScript
+
+# Tell git to not mess with line endings
+* binary
```

Comment:
> you want -text, not binary.

Agreed I'll close this as *we definitely don't want this merged by mistake* :rose: